skops/hub_utils/tests/test_hf_hub.py

```diff
@@ -271,6 +271,49 @@ def test_init(classifier_pickle, config_json):
         )
 
 
+def test_init_modelcard_creation(classifier_pickle, config_json):
+    # create a temp directory and delete it, we just need a unique name.
+    dir_path = tempfile.mkdtemp()
+    shutil.rmtree(dir_path)
+
+    version = metadata.version("scikit-learn")
+    init(
+        model=classifier_pickle,
+        requirements=[f'scikit-learn="{version}"'],
+        dst=dir_path,
+        task="tabular-classification",
+        data=iris.data,
+    )
+    _validate_folder(path=dir_path)
+    assert os.path.isfile(Path(dir_path) / "README.md")
+
+
+def test_override_init_modelcard(classifier_pickle, config_json):
+    # create a temp directory and delete it, we just need a unique name.
+    dir_path = tempfile.mkdtemp()
+    shutil.rmtree(dir_path)
+
+    version = metadata.version("scikit-learn")
+    init(
+        model=classifier_pickle,
+        requirements=[f'scikit-learn="{version}"'],
+        dst=dir_path,
+        task="tabular-classification",
+        data=iris.data,
+    )
+    _validate_folder(path=dir_path)
+    t0 = os.path.getmtime(Path(dir_path) / "README.md")
+
+    # override existent modelcard created by init
+    model = get_classifier()
+    model_card = card.Card(model, metadata=card.metadata_from_config(Path(dir_path)))
+    model_card.save(Path(dir_path) / "README.md")
+    t1 = os.path.getmtime(Path(dir_path) / "README.md")
+
+    # compare the times at which the files were last modified
+    assert t0 != t1
```

Comment:
I think instead of this we can render the content of model card and compare whether they're same or not, WDYT? @adrinjalali @jucamohedano

Reply:
I'm happy either way!

Reply:
We want to make sure content is modified so it would be nice to have the former.

Reply:
I made the change :)